onchain/build.nix

```diff
@@ -0,0 +1,25 @@
+{ ... }:
+{
+  imports = [ ./api/lbf/build.nix ];
+
+  perSystem = { config, ... }:
+    {
+      packages = {
+        remove-me-later-this-is-just-an-example-to-show-how-to-access-the-compiled-lbf-dens-schema =
+          config.packages.lbf-dens-plutarch;
+      };
+
+      devShells = {
+        # When writing the devShells, remember to run 
+        # ```
+        # config.devShellHook
+        # ```
+        # in the `shellHook`; and include 
+        # ```
+        # config.devShellTools
+        # ```
+        # in the `buildInputs`.
+      };
+
+    };
+}
```

Comment:
@gnumonik I'll leave this for you to fill in when you get the chance. If you're busy, and want a hand just let me know!

Reply:
I'm gonna merge this all and then do the nix setup myself (probably later today). Looks like you borrowed from LB so I'll try to see if I can steal most of the config from there.

Reply:
Indeed, almost everything is from LB.

Just a heads up, we're trying to move the more general nix project setup from LB to here https://github.com/mlabs-haskell/flake-lang.nix